test/Scripts.Integration.Test/SmokeTester.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.IO;
+using UnityEngine;
+
+public class SmokeTester : MonoBehaviour
+{
+    public void Start()
+    {
+        SmokeTest();
+    }
+
+    public static void SmokeTest()
+    {
+        // On Android we'll grep logcat for this string instead of relying on exit code:
+        Debug.Log("SMOKE TEST: PASS");
```

Comment:
Some other message that doesn't equal the actual smoke test (which tests Sentry).

```suggestion
        Debug.Log("LAUNCH SMOKE TEST: PASS");
```